src/components/VmUserMessages/index.js

```diff
@@ -85,10 +85,12 @@ class VmUserMessages extends React.Component {
         <NotificationDrawer hide={!this.state.show} expanded={this.state.expanded}>
           <NotificationDrawer.Title onCloseClick={this.handleToggle} onExpandClick={this.handleExpand} />
           <NotificationDrawer.PanelBody className={style['panel-body']}>
-            {messagesList}
+            <div className={style['drawer-pf-notifications-list']}>
+              {messagesList}
+            </div>
             { messagesCount > 0 &&
               <NotificationDrawer.PanelAction>
-                <NotificationDrawer.PanelActionLink data-toggle='clear-all'>
+                <NotificationDrawer.PanelActionLink data-toggle='clear-all' className={style['drawer-pf-action-clear-notification-list']}>
```

Comment:
Move the `className` up to the `PanelAction` instead of here on `PanelActionLink`.